acquisitions/src/main/resources/thunderjet/mod-finance/features/group-and-ledger-transfers-after-rollover.feature

```diff
@@ -0,0 +1,221 @@
+# For https://issues.folio.org/browse/MODFIN-299
+@parallel=false
+Feature: Group and ledger transfers after rollover
+
+  Background:
+    * print karate.info.scenarioName
+
+    * url baseUrl
+
+    * callonce login testUser
+    * def okapitokenUser = okapitoken
+
+    * def headersUser = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitokenUser)', 'Accept': 'application/json' }
+    * configure headers = headersUser
+
+    * callonce variables
+    * def fromYear = callonce getCurrentYear
+    * def toYear = parseInt(fromYear) + 1
+    * def fyId1 = callonce uuid1
+    * def fyId2 = callonce uuid2
+    * def ledgerId1 = callonce uuid3
+    * def ledgerId2 = callonce uuid4
+    * def fundId1 = callonce uuid5
+    * def fundId2 = callonce uuid6
+    * def fundId3 = callonce uuid7
+    * def f1Y1BudgetId = callonce uuid8
+    * def f1Y2BudgetId = callonce uuid9
+    * def f2Y1BudgetId = callonce uuid10
+    * def f2Y2BudgetId = callonce uuid11
+    * def f3Y1BudgetId = callonce uuid12
+    * def f3Y2BudgetId = callonce uuid13
+    * def groupId = callonce uuid14
+    * def rolloverId = callonce uuid15
+
+    * def createFiscalYear = read('classpath:thunderjet/mod-finance/reusable/createFiscalYear.feature')
```

Comment:
I see that you not declaring functions like createFund, createLedger, createBudget , declaring only  createFiscalYear .
So this is optional step?

Reply:
These are already defined by [karate-config.js](https://github.com/folio-org/folio-integration-tests/blob/master/acquisitions/src/main/resources/karate-config.js). Eventually we might want to add more there to lighten the features, or remove them to make all dependencies explicit and to be more consistent. I am not sure how it would affect performance, but this should be taken into account.